templates/hsdk-ngc-container/.devcontainer/post-start-command.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+printf "#################### .devcontainer/post-start-command.sh ####################\n"
+printf "#                                                                           #\n"
+printf "# Edit this file to add commands that should be run after the container is  #\n"
+printf "# started.                                                                  #\n"
+printf "#                                                                           #\n"
+printf "#############################################################################\n"
+
+
+echo Copying Holoscan SDK example applications to ~/examples...
+sudo mkdir -p /home/holoscan/.config/fish
+sudo cp -r /opt/nvidia/holoscan/examples /home/holoscan/examples
+
+echo Configuring /home/holoscan for $USER...
+sudo chown -R $USER:$USER /home/holoscan
```

Comment:
Is it possible that we are overwriting the permissions (owner) of the host file system, which may cause unintended behavior?

E.g., user's host system's USER ID is 1003 but we are changing host file system's owner to 1000 here.

I wonder if permission issues are properly handled by `common-debian.sh` script.

Reply:
The container is set to launch using 1000 by the `ghcr.io/devcontainers/features/common-utils:2` feature, similar to the `common-debian.sh` script.  

Also, the `/home/holoscan` directory is a Docker volume(not mounted to the host), named `holoscan-sdk-vscode-profile`. This volume is persisted so the user doesn't need to reinstall all the extensions every time the devcontainer launches.